tasks/install-with-package.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install Certbot.
+  package: name=certbot state=present
```

Comment:
12.04 could probably use from-source as well. But in my opinion, at least Ubuntu 16.04 should work out of the box, no? @geerlingguy